src/Command/CheckRequirements.php

```diff
@@ -0,0 +1,134 @@
+use Manala\Config\Requirement\Violation\RequirementViolation;
+use Manala\Config\Requirement\Violation\RequirementViolationLabelBuilder;
+use Manala\Config\Requirement\Violation\RequirementViolationList;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Checks if the host's environment meets Manala's requirements (Vagrant, Ansible, etc.).
+ *
+ * @author Xavier Roldo <xavier.roldo@elao.com>
+ */
+class CheckRequirements extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('check:requirements')
+            ->setDescription("Checks if your host's environment meets Manala's requirements (Vagrant, Ansible, etc.)")
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->addFormatter($output);
+
+        $violationList = new RequirementViolationList();
+        $requirementChecker = new RequirementChecker(
+            new HandlerFactoryResolver(),
+            new RequirementViolationLabelBuilder()
+        );
+
+        foreach($this->getRequirements() as $requirement) {
+            $output->writeln('Checking ' . $requirement->getName());
+            $requirementChecker->check($requirement, $violationList);
+        }
+
+        if (count($violationList) > 0) {
+            foreach ($violationList as $violation) {
+                $description = $this->getFormattedViolationDescription($violation);
+                $output->writeln($description);
+            }
+        }
+
+        if (!$violationList->containsRequiredViolations()) {
+            $output->writeln('Congratulations ! Everything seems OK.');
+            if ($violationList->containsRecommendedViolations()) {
+                $output->writeln('Yet, some recommendations have been emitted (see above).');
+            }
+        }
+
+    }
+
+    /**
+     * @return Requirement[]
+     */
+    private function getRequirements()
+    {
+        return [
+            new Requirement(
+                'vagrant',
+                Requirement::TYPE_BINARY,
+                RequirementLevel::REQUIRED,
+                '>= 1.8.0 < 1.8.5 || ^1.8.6', // /!\ Exclude vagrant 1.8.5 (Manala incompatible)
+                'See https://www.vagrantup.com/downloads.html'
+            ),
+            new Requirement(
+                'landrush',
+                Requirement::TYPE_VAGRANT_PLUGIN,
+                RequirementLevel::REQUIRED,
+                '^0.18',
+                'See https://github.com/vagrant-landrush/landrush'
+            ),
+            new Requirement(
+                'ansible',
+                Requirement::TYPE_BINARY,
+                RequirementLevel::RECOMMENDED,
+                '^2.0.0',
```

Comment:
Same as landrush, we should use the last available release, aka. 2.1.1
See https://github.com/ansible/ansible/releases